Autodesk/synthesis
# Bare Minimum Analytics for Upcoming Release

Based on team discussions and the upcoming build, I incorporated some bare minimum analytics which includes:

- Main UI buttons (e.g. MoveArrows, DeleteEntity, AddEnvironment, AddEntity, etc.)
- Exporter data (whether the model is from Fusion or Inventor)
- A clean Google Analytics project to track _only_ data from our new modernized Synthesis.

Note: Some API changes were made for the purpose of analytics.

**Testing**
1. Log into the autodeskbxd@gmail.com account via Google Analytics.
2. In the Google Analytics home dashboard, click on the new project "Synthesis App".
3. Navigate to the "Realtime" data and click "Events". 
4. Run Synthesis to see live event updates with basic button clicking and the fulfilled exporter data.

**Improvements**
Due to the decision of incorporating just some basic analytics, there are some updates I will be adding in a follow-up PR to include things like timing variables (they don't have realtime data so it takes a long time to test) and real analytics vs. development analytics (an option to disable analytics while doing development testing.) These improvements will be noted in #643 .
